lib/modules/versioning/pep440/range.ts

```diff
@@ -98,6 +98,10 @@ interface Range {
   version: string;
 }
 
+export function getPinnedValue(newVersion: string): string {
+  return '==' + newVersion;
```

Comment:
Just to be consistent as we use string templates in most places.
```suggestion
  return `==${newVersion}`;
```